packages/rum-core/src/common/apm-server.js

```diff
@@ -239,7 +239,7 @@ class ApmServer {
   }
 
   ndjsonMetricsets(metricsets) {
-    const timestamp = Date.now()
+    const timestamp = Date.now() * 1000
```

Comment:
this was missed when changing the implementation from `microtime`, has to be in microseconds for the UI to display the breakdown.